### ISSUE TITLE
how do i build this on windows 11

### ISSUE THREAD

**Issue by jbmanlol** [Herringway/ebbinex#7]
its broken on windows 11 so can somebody fix it?

**Comment by Herringway** [comment]
What's broken about it? I am using windows 11 to develop it, and it runs just fine for me.

**Comment by jbmanlol** [comment]
like when i drag the windows exe it opens and then disappears

**Comment by Herringway** [comment]
Yes, that's normal for a command line program. Run it from a command prompt.

**Comment by jbmanlol** [comment]
give me the command for it i need help

**Comment by jbmanlol** [comment]
ok but how i use it like with the command prompt it just gives me a error

**Comment by Herringway** [comment]
If you're building ebsrc, follow the directions there. If you aren't, you are most certainly using the wrong program for whatever it is you're trying to do.

**Comment by jbmanlol** [comment]
i am trying to use your wip earthbound pc port

**Comment by Herringway** [comment]
It does not use this tool.

**Comment by jbmanlol** [comment]
how do you extract then

**Comment by Herringway** [comment]
https://github.com/Herringway/earthbound/issues/81#issuecomment-1515546708
